web/package.json

```diff
@@ -22,6 +22,7 @@
     "prebuild": "pnpm run create-generated-files",
     "cy:component": "cypress open --component -b electron",
     "cy:e2e": "cypress open --e2e -b electron",
+    "cy:test-all": "cypress run && cypress run --component",
```

Comment:
should this also use the electron browser as the cy:component script or does that not matter?

Reply:
Electron is the default :)